controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/routing/responses/endpoint/endpoints.json

```diff
@@ -0,0 +1,31 @@
+{
+  "endpoints": [
+    {
+      "endpoint": "default",
```

Comment:
Consider calling this `name` or `endpointName`.

Reply:
`name`, agreed.